prog/vm/github_runner.rb

```diff
@@ -155,6 +155,9 @@ def before_run
   end
 
   label def register_runner
+    # Do not register runner if the runner-script service is already running.
```

Comment:
Why would that happen? wait ensures that the service is not running before redirecting to here.

Reply:
If the respirate crashes after the "./actions-runner/run.sh" script but before it hops, it fails in the subsequent run. We investigate a page like that with you.

Reply:
However, it's extremely rare. I'm uncertain if it's worthwhile to incorporate an additional SSH call for all runs. In 99.99% of cases, it returns "dead".

Reply:
I think it is OK to have ssh call in this case. We don't do it frequently.